src/handlers/list.ts

```diff
@@ -1,10 +1,28 @@
+import { readFile } from "node:fs/promises";
+import { Task } from "../models/Task";
+
 export namespace ListHandler {
   export type Args = {
     all: boolean;
   };
 
+  const taskData = "data/db.json";
+
   export const handle = async (args: Args): Promise<void> => {
-    // TODO
-    throw new Error("Not yet implemented");
+    const data = await readFile(taskData);
+    const stringContent = data.toString();
+    const taskObjects: Array<Task> = JSON.parse(stringContent);
```

Comment:
What happens if the file is damaged? We may need to handle errors here

Reply:
You are right! I fixed and pushed the changes so that please check!